django/cantusdb_project/main_app/views/chant.py

```diff
@@ -1,3 +1,4 @@
+from turtle import position
```

Comment:
wayward import

Reply:
personally, I think we should do away with the position field and just use the one from `turtle`

Reply:
fine. done.